packages/graphql_server/src/api/githubApi.ts

```diff
@@ -65,3 +71,69 @@ export async function getUserInfo(query: string, authToken: string): Promise<Git
   )
   return response.data.data.user
 }
+
+/** Retrieves the contributor count for a GitHub repository.
+ * This may be smaller than the count on the Github page because only contributors that
+ * committed into the main branch are being counted
+ * @param owner - The owner of the GitHub repository.
+ * @param repo - The name of the GitHub repository.
+ * @param authToken - Github API token
+ * @returns A Promise that resolves to the total unique contributor count; returns 0 on error
+ */
+export async function getContributorCount(
+  owner: string,
+  repo: string,
+  authToken: string
+): Promise<number> {
+  const query = `
+    query($owner: String!, $repo: String!) {
+      repository(owner: $owner, name: $repo) {
+        defaultBranchRef {
+          target {
+            ... on Commit {
+              history {
+                totalCount
+                edges {
+                  node {
+                    author {
+                      user {
+                        login
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  `
+
+  const variables = {
+    owner,
+    repo
+  }
+
+  try {
```

Comment:
don't catch errors, so we can catch them in the GraphQL resolvers and give appropriate feedback and logging where due